### PR TITLE
Update nf-core/isoseq to version 1.1.2

### PR DESCRIPTION
**Pull request by sguizard**
## PR checklist

The pipeline has been updated to version 2.6 of files templates. 
The isoseq modules have been updated. 
The update of uLTRA fix an issue. In some bam files, the reference sequence ID was wrong. 
The --capped option is not hard coded anymore. 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/isoseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/isoseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules.json [hunk 1]
```diff
@@ -2,60 +2,17 @@
     "name": "nf-core/isoseq",
     "homePage": "https://github.com/nf-core/isoseq",
     "repos": {
-        "nf-core/modules": {
-            "git_url": "https://github.com/nf-core/modules.git",
+        "https://github.com/nf-core/modules.git": {
```
Comment (JoseEspinosa) [inline modules.json:5]: Think there is a problem with this file, I think you probably took it from the template directly

**Review by JoseEspinosa — DISMISSED** [review]
LGTM! Though I think is there a problem with the `modules.json` file

File: modules.json (hunk 1 above)
Comment (sguizard) [reply modules.json:5]: I reinstalled all modules to fix the modules.json file. 
Thanks for pointing this issue.

**Comment by maxulysse** [comment]
Looks ok to me too, only issue I can see is it's an old TEMPLATE.
If we can merge https://github.com/nf-core/isoseq/pull/12 before this one, it'll be perfect

File: modules/local/samplesheet_check.nf [hunk 2]
```diff
@@ -1,8 +1,8 @@
 process SAMPLESHEET_CHECK {
     tag "$samplesheet"
-    label 'process_low'
+    label 'process_single'
 
-    conda (params.enable_conda ? "conda-forge::python=3.8.3" : null)
+    conda "conda-forge::python=3.8.3"
```
Comment (maxulysse) [inline modules/local/samplesheet_check.nf:5]: Need to do the same in https://github.com/nf-core/isoseq/blob/dev/modules/local/gstama/filelist/main.nf

Comment (sguizard) [reply modules/local/samplesheet_check.nf:5]: Updated, Thanks for pointing this.

File: workflows/isoseq.nf [hunk 3]
```diff
@@ -133,10 +137,18 @@ workflow ISOSEQ {
...
         ch_tcollapse,
         Channel.value("capped"),
         Channel.value("1,1,1"))
+    }
+    else {
+        GSTAMA_FILELIST( // Generate the filelist file needed by TAMA merge
+        ch_tcollapse,
+        Channel.value("no_cap"),
+        Channel.value("1,1,1"))
+    }
```
Comment (maxulysse) [inline workflows/isoseq.nf:151]:
```suggestion
    cap_value = params.capped == true ? Channel.value("capped") : Channel.value("no_cap")

    GSTAMA_FILELIST( // Generate the filelist file needed by TAMA merge
    ch_tcollapse,
    cap_value,
    Channel.value("1,1,1"))
```

**Review by maxulysse — APPROVED** [review]
LGTM

File: workflows/isoseq.nf (hunk 3 above)
Comment (maxulysse) [reply workflows/isoseq.nf:151]: I let you choose if you want to rewrite that part or not.

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,25 @@
...
+## v1.1.2 - Gray Eagle [11/01/2023]
+
+### `Added`
+
+- Update module pbccs (v6.4.0)
+- Update module isoseq refine (v3.8.1)
+- Update module lima (v2.6.0)
+- Update module uLTRA (v0.0.4.2). Fix [issue #17](https://github.com/ksahlin/ultra/issues/17). Thanks to [Husen M. Umer](https://github.com/husensofteng).
+- Zenodo DOI
+- Update to template v2.7.2
```
Comment (d4straub) [inline CHANGELOG.md:15]: In many pipelines the PR that made the change is listed here, just as future suggestion.

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,25 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.2 - Gray Eagle [11/01/2023]
+
+### `Added`
+
+- Update module pbccs (v6.4.0)
+- Update module isoseq refine (v3.8.1)
+- Update module lima (v2.6.0)
```
Comment (d4straub) [inline CHANGELOG.md:12]: This looks better and is more informative in a table format, e.g. as in https://github.com/nf-core/ampliseq/blob/master/CHANGELOG.md#dependencies-1, imho. Just as a comment.

**Review by d4straub — APPROVED** [review]
Looks good to me.

File: workflows/isoseq.nf (hunk 3 above)
Comment (sguizard) [reply workflows/isoseq.nf]: Thanks for the suggestion!

File: CHANGELOG.md (hunk 5 above)
Comment (sguizard) [reply CHANGELOG.md]: Thanks for this suggestion, already I added it.

File: workflows/isoseq.nf [hunk 6]
```diff
@@ -133,10 +137,12 @@ workflow ISOSEQ {
...
 
+    cap_value = params.capped == true ? Channel.value("capped") : Channel.value("no_cap")
+
     GSTAMA_FILELIST( // Generate the filelist file needed by TAMA merge
-        ch_tcollapse,
-        Channel.value("capped"),
-        Channel.value("1,1,1"))
+    ch_tcollapse,
+    cap_value,
+    Channel.value("1,1,1"))
```
Comment (maxulysse) [inline workflows/isoseq.nf:145]:
```suggestion
        ch_tcollapse,
        cap_value,
        Channel.value("1,1,1"))
```
We lost indentation

**Pull request merged**